oslo_vmware/rw_handles.py

```diff
@@ -177,6 +177,32 @@ def read(self, chunk_size):
         """
         raise NotImplementedError()
 
+    def tell(self):
+        """Get the position of File Handle
+
+        :return: position
+        """
+        raise NotImplementedError()
+
+    def fileno(self):
+        """returns the integer file descriptor
+        by default this is not supported and raises IOError
+        """
+        raise IOError()
+
+    def seek(self, offset):
+        """sets the file's current position at the offset
+
+        :param offset: offset
+        """
+        pass
```

Comment:
I think this needs more documentation why this would be ok behavior. I know we can't seek, but why can't we raise an exception here, then? Why do we need it in the first place? I think that's pretty unexpected to have a seek "work" but actually it didn't work.

Reply:
Perhaps we can raise an exception here, and overwrite it down in VmdkWriteHandle with a pass.
We can't seek, however we should to be tolerant to it because there are places where seek exceptions are not handled: https://github.com/sapcc/cinder/blob/stable/queens-m3/cinder/backup/chunkeddriver.py#L665

Reply:
It seems that `.seek()` does not throw errors in python (2 and 3), excepting if offset is negative.
However, python3's seek returns an integer value.

Reply:
My point is, that we currently know what we want to do with that change, but in the future, someone else might call the code and actually expect seeking to happen, thus resulting in broken files.

Reply:
seek operations don't typically return anything.   So I wouldn't recommend raising an exception here.   I would think the child classes would do real work for seek.   It makes sense for the child class to implement the real seek function.